$NetBSD: patch-CMakeLists.txt,v 1.1 2025/10/22 12:07:56 gdt Exp $

Cope with boost deprecations.

Sent upstream by email 20251021, because their bugtracker both
requires and prohibits registrations.

--- CMakeLists.txt.orig	2024-05-15 17:40:15.000000000 +0000
+++ CMakeLists.txt
@@ -119,7 +119,7 @@ endif()
 ########################################################################
 # Find boost
 ########################################################################
-find_package(Boost "1.65" REQUIRED chrono thread system)
+find_package(Boost "1.65" REQUIRED chrono thread)
 
 if(NOT Boost_FOUND)
     message(FATAL_ERROR "Boost required to compile osmosdr")
